backend/spec/requests/books_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+require './spec/helpers/api_helpers'
+
+RSpec.configure do |config|
+  config.include ApiHelpers, type: :request
+end
```

Comment:
Move this inside the `rails_helper` file. Any configurations that affect `Rspec` should be put there. 

Else you risk making it hard for you to know which file is defining a specific rspec configuration.